python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/ptp.schema.yml

```
@@ -166,3 +166,15 @@ keys:
                     max: 255
                     convert_types:
                     - str
+      free_running:
+        type: dict
+        keys:
+          enabled:
+            type: bool
```

This should be required key
```suggestion
          enabled:
            type: bool
            required: true
```

---

updated